app/classifier/tasks/survey/choice.cjsx

```diff
@@ -44,15 +44,19 @@ ImageFlipper = React.createClass
 module.exports = React.createClass
   displayName: 'Choice'
 
+  propTypes:
+    annotation: React.PropTypes.object.isRequired
```

Comment:
It is and I will add it to the type check. Technically `annotation` here was a misnomer, because it refers to only the `value` of an annotation. Understanding the variable names was part of the challenge for me.